lib/src/app/app.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:async';
 import 'package:viam_sdk/src/gen/google/protobuf/struct.pb.dart';
 
 import '../gen/app/v1/app.pbgrpc.dart';
+import '../gen/common/v1/common.pb.dart';
```

Comment:
@njooma / @clintpurser my best guess at handling [the move of `LogEntry` to the common namespace](https://github.com/viamrobotics/api/pull/435).